### ISSUE TITLE
APIByCategory page is not available in wiki

### ISSUE THREAD

**Issue by Cacodemon345** [libsdl-org/SDL#4073]
I can't access the APIByCategory page on the wiki due to it being too large to render.

**Comment by erikyuzwa** [comment]
yeah same - I don't immediately know what the proper "fix" is - unless it's restructuring the entire page to make it more navigation-snappy? 

I'm offering to dig into this as well @icculus

**Comment by Ezall2001** [comment]
same here can't access the page. I really need it for my uni project. is there like a hack around or a quick fix for this?

**Comment by Lokathor** [comment]
you can clone the wiki to a local copy and then open it up in whatever.

**Comment by erikyuzwa** [comment]
@Ezall2001 - you can try the Wayback Machine - https://web.archive.org/web/20201129211815/https://wiki.libsdl.org/CategoryAPI

**Comment by Ezall2001** [comment]
@erikyuzwa I love you bro. you saved my day.

**Comment by TDoc81** [comment]
Is anybody going to fix this? The internet archive version is broken for me and there's no other mirror for the acual wiki. I've managed to build a doxygen of the source but it's not very navigable and frustrating to use.

**Comment by icculus** [comment]
> Is anybody going to fix this?

We're working on it.

**Comment by icculus** [comment]
For the time being, https://wiki.libsdl.org/ is serving the original content again, while we figure out what we're doing next.

**Comment by icculus** [comment]
This is resolved now (wiki.libsdl.org is running new software, and this page works on it).
